src/visitor.rs

```diff
@@ -77,12 +83,19 @@ impl<'a, R: FileResolver> VisitMut for Visitor<'a, R> {
             let mut visitor = Visitor::new(
                 &first_candidate,
                 false,
+                self.annotate_paths,
                 self.error_log.as_mut().map(|v| &mut **v),
                 self.resolver,
             );
 
             match visitor.visit() {
                 Ok(syn::File { attrs, items, .. }) => {
+                    if self.annotate_paths {
```

Comment:
To the comment above, I wonder if we should check here that no attribute in the input is using our path, returning an error if it is?